operate/services/manage.py

```diff
@@ -106,6 +106,9 @@
 URI_HASH_POSITION = 7
 IPFS_GATEWAY = "https://gateway.autonolas.tech/ipfs/"
 DEFAULT_TOPUP_THRESHOLD = 0.5
+# At the moment, we only support running one agent per service locally on a machine.
+# If multiple agents are provided in the service.yaml file, only the 0th index config will be used.
+SUPPORTED_NUM_OF_AGENTS = 1
```

Comment:
Please refactor to something more meaningful, e.g.,
`NUM_LOCAL_AGENT_INSTANCES`